src/lib/hl7v2/hl7v2.js

```diff
@@ -172,5 +209,3 @@ function parseInvalidAccess(parsedMsg) {
 }
 
 module.exports.parseHL7v2 = parseHL7v2;
```

Comment:
It looks like this is only used in spec files now. Is there a way to change the tests so this doesn't need to be exported?

Reply:
Right. Is it okay to take it in next change if not a big concern (to unblock other stuff)?